.gitignore

```diff
@@ -0,0 +1 @@
+db.rb
```

Comment:
Would be good to provide a sample file, like our other secret files have in other projects.